src/mem/cache/prefetch/stride.hh

```diff
@@ -105,6 +105,8 @@ class Stride : public Queued
 
     const int degree;
 
+    const int distance;
```

Comment:
Nit: Could you document what the distance means, so that it is easier for students to understand this prefetcher? Feel free to copy what you wrote in the python file.

Reply:
Done.